torch/jit/_recursive.py

```diff
@@ -379,6 +379,7 @@ def create_script_module_impl(nn_module, concrete_type, stubs_fn):
         stubs_fn:  Lambda that takes an nn.Module and generates a list of ScriptMethodStubs to compile.
     """
     cpp_module = torch._C._create_module_with_type(concrete_type.jit_type)
+    nn_module = nn_module.__prepare_scriptable__() if hasattr(nn_module, '__prepare_scriptable__') else nn_module
```

Comment:
Since we call `call_prepare_scriptable_func` before `create_script_module` in `_script.py` file, we don't need to call `__prepare_scriptable__` func here again. I think once we remove this line, the CI tests should pass. Crossed fingers.